linux/LinuxProcessList.c

```diff
@@ -1436,6 +1430,13 @@ static bool isOlderThan(const ProcessList* pl, const Process* proc, unsigned int
    return realtime - proc->starttime_ctime > seconds;
 }
 
+static bool isContainerOrVMSlice(const char* cgroup) {
+   if (String_startsWith(cgroup, "/user") || String_startsWith(cgroup, "/system"))
```

Comment:
I'm not sure this works as expected. This won't catch containers on Proxmox, but might accidently match non-containers otherwise.